lib/openxml/docx/elements/grid_column.rb

```diff
@@ -0,0 +1,18 @@
+module OpenXml
+  module Docx
+    module Elements
+      class GridColumn < Element
+        tag :gridColumn
+
+        attribute :w, expects: :valid_number
+
+      private
+
+        def valid_number(value)
+          value =~ /^[0-9]+$/
+        end
```

Comment:
No, it really should just use the validation built-in to `AttributeBuilder`: `expects: :integer`.